src/psyclone/tests/domain/lfric/lfric_loop_bounds_test.py

```diff
@@ -32,6 +32,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
 # Author: A. R. Porter, STFC Daresbury Lab
+# Modified by L. Turner, Met Office
```

Comment:
I would suggest `# Modified: L. Turner, Met Office` for consistency.

Reply:
done fd86da3